src/tools/extract.cpp

```diff
@@ -67,10 +67,10 @@ return_code parseArguments(int argc,
                                   &extractor_config.location_dependent_data_paths)
                                   ->composing(),
                               "GeoJSON files with location-dependent data")(
-        "use-locations-cache",
-        boost::program_options::value<bool>(&extractor_config.use_locations_cache)
-            ->implicit_value(true)
-            ->default_value(extractor_config.use_locations_cache),
+        "disable-locations-cache",
```

Comment:
Minor semantics: If we need to change this anyway, can we rename it to `disable-location-cache` (drop the `s` on `locations`)? (would also need to update changelog).

Reason: noun adjuncts in English are *usually* singular, unless there is a lexical restrictions (https://en.wikipedia.org/wiki/Noun_adjunct#English).